book/10-git-internals/sections/objects.asc

```diff
@@ -381,6 +381,14 @@ You can calculate the SHA-1 value of a string in Ruby by including the SHA1 dige
 => "bd9dbf5aae1a3862dd1526723246b20206e5fc37"
 ----
 
+Let's validate the hash with `git hash-object`. It shows that they are identical. Note that `echo` will add a trailing newline character in string, which is not we want in this case. `echo -n` can prevent from adding a trailing newline character.
+
+[source,console]
+----
+>> $ echo -n "what is up, doc?" | git hash-object --stdin
+=> bd9dbf5aae1a3862dd1526723246b20206e5fc37
+----
```

Comment:
Here you've got a shell sample marked up like it was typed into `irb`, which won't work. Could you remove the `>>` and `=>` prefixes?